pkg/cli/cli.go

```diff
@@ -25,6 +25,13 @@ func Run(configPath string, inputPath string, outputPath string) error {
 			obfuscators = append(obfuscators, obfuscator.NewKeywordsObfuscator(o.Replacement))
 		case schema.ObfuscateTypeMAC:
 			obfuscators = append(obfuscators, obfuscator.NewMacAddressObfuscator())
+		case schema.ObfuscateTypeDomain:
+			// The schema does not make the default value for the TLDs a string array instead an interface array.
+			var configTLDs []string
+			for _, tld := range config.Config.TopLevelDomains {
+				configTLDs = append(configTLDs, tld.(string))
+			}
+			obfuscators = append(obfuscators, obfuscator.NewDomainObfuscator(configTLDs))
```

Comment:
if you want to rebase over #16 then you get that for free. The generator should generate the string types properly now. 

Let me know what you prefer, I can also fix it in #16 after we merge this PR